ios/predictions.md

```diff
@@ -131,10 +131,7 @@ Add the dependencies to the `Podfile`:
 ```ruby
 target :'YOUR-APP-NAME' do
 	use_frameworks!
-
-	pod 'Amplify', '~> 1.0.0'
-	pod 'AmplifyPlugins/AWSPluginsCore', '~> 1.0.0'  # pod 'AWSPluginsCore', '~> 1.0.0
-	pod 'AmplifyPlugins/AWSPredictionsPlugin', '~> 1.0.0'
+	pod 'AWSPredictionsPlugin'
```

Comment:
Do we not need the "Amplify" and core plugin ?

Reply:
We dont, I was able to make it work without that.

Reply:
just confirmed for AWSAPIPlugin